### PR TITLE
Adding a sentence to the index to point people towards the "activate …

### PR DESCRIPTION
**Pull request by 09jvilla**
…license" page if they want to complete installation of Grafana Enterprise (since you don't just need the enterprise binary, you also need to install your license file). Also made a minor change to the two sentences before to make them more consistent. 

I closed my original PR (https://github.com/grafana/grafana/pull/32564) because when I rebased it looked like I had merged a bunch of additional PRs that weren't mine. (Probably some mistaken I made in the ordering of git pull from master / git rebase / git push).

### CODE REVIEW

File: docs/sources/installation/_index.md [hunk 1]
```diff
@@ -18,5 +18,7 @@ This section discusses the hardware and software requirements as well as the pro
 - [Run Docker image]({{< relref "docker" >}})
 - [Deploy Grafana on Kubernetes]({{< relref "kubernetes" >}})
 
-Refer to [Upgrade Grafana]({{< relref "upgrading.md" >}}) topic. If you need to restart Grafana, refer to the [Restart Grafana]({{< relref "Restart-grafana.md" >}})
-topic for detailed information.
+Refer to [Upgrade Grafana]({{< relref "upgrading.md" >}}) for how to upgrade Grafana.
```
Comment (osg-grafana) [inline docs/sources/installation/_index.md:21]:
```suggestion
To upgrade Grafana, refer to [Upgrade Grafana]({{< relref "upgrading.md" >}}).
```

File: docs/sources/installation/_index.md [hunk 2]
```diff
@@ -18,5 +18,7 @@ This section discusses the hardware and software requirements as well as the pro
...
+Refer to [Restart Grafana]({{< relref "Restart-grafana.md" >}}) for how to restart Grafana.
```
Comment (osg-grafana) [inline docs/sources/installation/_index.md:22]:
```suggestion
To restart Grafana, refer to [Restart Grafana]({{< relref "Restart-grafana.md" >}}).
```

**Review by osg-grafana — CHANGES_REQUESTED** [review]

**Review by osg-grafana — APPROVED** [review]
Approving copy changes.

File: docs/sources/installation/_index.md [hunk 3]
```diff
@@ -18,5 +18,7 @@ This section discusses the hardware and software requirements as well as the pro
 - [Run Docker image]({{< relref "docker" >}})
 - [Deploy Grafana on Kubernetes]({{< relref "kubernetes" >}})
 
-Refer to [Upgrade Grafana]({{< relref "upgrading.md" >}}) topic. If you need to restart Grafana, refer to the [Restart Grafana]({{< relref "Restart-grafana.md" >}})
-topic for detailed information.
+To upgrade Grafana, refer to [Upgrade Grafana]({{< relref "upgrading.md" >}}).
```
Comment (achatterjee-grafana) [inline docs/sources/installation/_index.md:21]:
```suggestion
For upgrade instructions, refer to [Upgrade Grafana]({{< relref "upgrading.md" >}}).
```

**Review by achatterjee-grafana — APPROVED** [review]
Added more minor copy-edit suggestion. Otherwise LGTM

File: docs/sources/installation/_index.md [hunk 4]
```diff
@@ -18,5 +18,7 @@ This section discusses the hardware and software requirements as well as the pro
...
+To restart Grafana, refer to [Restart Grafana]({{< relref "Restart-grafana.md" >}}).
```
Comment (achatterjee-grafana) [inline docs/sources/installation/_index.md:22]:
```suggestion
To restart Grafana, refer to [Restart Grafana]({{< relref "restart-grafana.md" >}}).
```

**Review by achatterjee-grafana — COMMENTED** [review]
Fixed an incorrect relref.

**Pull request merged**